test/endpoint_tests/test_graphql.py

```diff
@@ -2,54 +2,87 @@
 from boxwise_flask.models import Camps, Cms_Users
 
 
+def get_base_from_graphql(id, base_query):
+    return [x for x in base_query if x["id"] == id][0]
+
+
 def test_all_bases(client):
     """Verify allBases GraphQL query endpoint"""
+    camps = [
+        {"id": 1, "name": "oak-tree", "organisation_id": 1, "currencyname": "pound"},
+        {"id": 2, "name": "chicken", "organisation_id": 1, "currencyname": "peanuts"},
+        {"id": 3, "name": "sofa", "organisation_id": 1, "currencyname": "candles"},
+    ]
 
     db.connect_db()
-    Camps.create(id=1, organisation_id=1, name="some text1", currencyname="hello")
-    Camps.create(id=2, organisation_id=1, name="some text1", currencyname="hello")
-    Camps.create(id=3, organisation_id=1, name="some text1", currencyname="hello")
-    Camps.create(id=4, organisation_id=1, name="some text1", currencyname="hello")
-    Camps.create(id=5, organisation_id=1, name="some text1", currencyname="hello")
+
+    for camp in camps:
+        Camps.create(
+            id=camp["id"],
+            organisation_id=camp["organisation_id"],
+            name=camp["name"],
+            currencyname=camp["currencyname"],
+        )
+
     db.close_db(None)
 
-    graph_ql_query_string = """query { \
-                allBases { \
-                    id \
-                    organisation_id \
-                    name \
-                } \
+    graph_ql_query_string = """query {
+                allBases {
+                    id
+                    organisation_id
+                    name
+                    currencyname
+                }
             }"""
 
     data = {"query": graph_ql_query_string}
     response_data = client.post("/graphql", json=data)
     assert response_data.status_code == 200
-    assert response_data.json["data"]["allBases"][0]["id"] == 1
+    all_bases = response_data.json["data"]["allBases"]
+    for camp in camps:
+        camper = get_base_from_graphql(camp["id"], all_bases)
```

Comment:
Maybe `expected_camp` instead of `camp`, and `actual_camp` instead of `camper`?
Does `assert expected_camp == actual_camp` work, too?